hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -101,12 +102,13 @@ public void dispatchHandle(
             @NonNull final HederaFunctionality function,
             @NonNull final TransactionBody txn,
             @NonNull final WritableStoreFactory writableStoreFactory) {
+        final var topicStore = writableStoreFactory.createTopicStore();
         switch (function) {
             case CONSENSUS_CREATE_TOPIC -> dispatchConsensusCreateTopic(
-                    txn.consensusCreateTopic().orElseThrow(), writableStoreFactory, usageLimits);
-            case CONSENSUS_UPDATE_TOPIC -> dispatchConsensusUpdateTopic(txn.consensusUpdateTopic().orElseThrow());
-            case CONSENSUS_DELETE_TOPIC -> dispatchConsensusDeleteTopic(txn.consensusDeleteTopic().orElseThrow());
-            case CONSENSUS_SUBMIT_MESSAGE -> dispatchConsensusSubmitMessage(txn.consensusSubmitMessage().orElseThrow());
+                    txn.getConsensusCreateTopic().orElseThrow(), topicStore, usageLimits);
+            case CONSENSUS_UPDATE_TOPIC -> dispatchConsensusUpdateTopic(txn.getConsensusUpdateTopic().orElseThrow(), topicStore);
+            case CONSENSUS_DELETE_TOPIC -> dispatchConsensusDeleteTopic(txn.getConsensusDeleteTopic().orElseThrow(), topicStore);
+            case CONSENSUS_SUBMIT_MESSAGE -> dispatchConsensusSubmitMessage(txn, topicStore);
```

Comment:
It looks like these are reverting PBJ changes (getFoo vs. foo method calls)

Reply:
yeah agreed ! This should be reverted ?

Reply:
Fixed.